store/storetest/mocks/TeamStore.go

```diff
@@ -4,9 +4,9 @@
 
```

Comment:
Looks like this whole file was accidentally included from a previous iteration.

Reply:
Fixed